pkg/restore/client.go

```diff
@@ -216,6 +223,23 @@ func (rc *Client) CreateTables(
 	return rewriteRules, newTables, nil
 }
 
+func (rc *Client) setSpeedLimit() error {
+	if !rc.hasSpeedLimited {
```

Comment:
```suggestion
	if !rc.hasSpeedLimited && rc.rateLimit != 0 {
```

Should check whether it needs a ratelimit.

Reply:
In fact even set 0 is considered unlimited。